### PR TITLE
Increase code coverage 4

### PR DESCRIPTION
**Pull request by RoyEJohnson**
[DISCO-355]

[DISCO-355]: https://openstax.atlassian.net/browse/DISCO-355?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/helpers/use-document-head.ts [hunk 1]
```diff
@@ -117,6 +93,9 @@ export function setPageTitleAndDescriptionFromBookData(data: BookData = {}) {
         ? htmlToText(data.description)
         : '';
 
+    if (data.bookSubjects && data.bookCategories) {
+        console.info('*** BOOK SUBJECTS AND CATEGORIES FOUND!!!! ');
```
Comment (jivey) [inline src/app/helpers/use-document-head.ts:97]: Not sure if debug...

File: test/src/helpers/use-document-head.test.tsx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+            </MemoryRouter>
+        );
+        expect(
+            document.head.querySelector('link')?.getAttribute('href')
+        ).toContain(canonicalPath);
+    });
+    it('handles fails gracefully if there is no title', () => {
+        const titleEl = document.head.querySelector('title');
+
+        document.head.removeChild(titleEl as Node);
```
Comment (jivey) [inline test/src/helpers/use-document-head.test.tsx:241]: I see the comment above... maybe a guard here though so it doesn't error if the test or describe block is focused?

File: test/src/components/shell/use-link-handler.test.tsx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        jest.spyOn(linkHelper, 'validUrlClick').mockReturnValue(false);
+
+        render(
+            <MemoryRouter initialEntries={['/']}>
+                <Component />
+            </MemoryRouter>
+        );
+        await user.click(screen.getByRole('link'));
+        expect(notPrevented).toBeCalledWith(true);
+        jest.resetAllMocks();
```
Comment (jivey) [inline test/src/components/shell/use-link-handler.test.tsx:51]: `afterEach` on the parent describe block could work for all the reset calls, I think

**Review by jivey — APPROVED** [review]
Left some fairly minor suggestions otherwise looks good

File: src/app/helpers/use-document-head.ts (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/helpers/use-document-head.ts:97]: Whoops.

**Pull request merged**